packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as React from 'react';
+import { DonutChart } from '../DonutChart/index';
+import { VerticalStackedBarChart } from '../VerticalStackedBarChart/index';
+import {
+  transformPlotlyJsonToDonutProps,
+  transformPlotlyJsonToColumnProps,
+  transformPlotlyJsonToScatterChartProps,
+  transformPlotlyJsonToHorizontalBarWithAxisProps,
+  isDateArray,
+  isNumberArray,
+  transformPlotlyJsonToHeatmapProps,
+  transformPlotlyJsonToSankeyProps,
+  transformPlotlyJsonToGaugeProps,
+} from './PlotlySchemaAdapter';
+import { LineChart } from '../LineChart/index';
+import { HorizontalBarChartWithAxis } from '../HorizontalBarChartWithAxis/index';
+import { AreaChart } from '../AreaChart/index';
+import { HeatMapChart } from '../HeatMapChart/index';
+import { SankeyChart } from '../SankeyChart/SankeyChart';
+import { GaugeChart } from '../GaugeChart/index';
+
+/**
+ * DeclarativeChart props.
+ * {@docCategory DeclarativeChart}
+ */
+export interface DeclarativeChartProps extends React.RefAttributes<HTMLDivElement> {
+  /**
+   * The schema representing the chart
+   */
+  chartSchema: any;
+}
+
+/**
+ * DeclarativeChart component.
+ * {@docCategory DeclarativeChart}
+ */
+export const DeclarativeChart: React.FunctionComponent<DeclarativeChartProps> = React.forwardRef<
```

Comment:
This PR should have only negative value changes. And not declarative chart changes.

Reply:
Okay. Closing this PR.

Reply:
New PR: [33377](https://github.com/microsoft/fluentui/pull/33377)